core/src/commonMain/kotlin/com/ustadmobile/core/controller/ClazzListPresenter.kt

```diff
@@ -42,13 +32,13 @@ class ClazzListPresenter(context: Any, arguments: Map<String, String>, view: Cla
         clazzList2ItemListener.listViewMode = mListMode
 
         loggedInPersonUid = accountManager.activeAccount.personUid
-        view.sortOptions = SortOrder.values().toList().map { ClazzListSortOption(it, context) }
+        selectedSortOption = SORT_OPTIONS[0]
         updateList()
     }
 
-    private fun updateList() {
-        view.list = repo.clazzDao.findClazzesWithPermission(searchQuery,
-                loggedInPersonUid, filterExcludeMembersOfSchool, currentSortOrder.sortOrderCode)
+    private fun updateList(searchText: String? = null) {
+        view.list = repo.clazzDao.findClazzesWithPermission(if(searchText.isNullOrEmpty()) "%%" else "%${searchText}%",
```

Comment:
I think we should have a string extension function instead of having so many if statements: e.g.

```
fun String?.toQueryLikeParam() = if(isNullOrEmpty()) "%" else "%this%"
```